sphinx-docs/installguide/install_all.rst

```diff
@@ -0,0 +1,206 @@
+Windows Installation
+====================
+
+1. Download the KA Lite `Windows <https://learningequality.org/r/windows-installer-latest>`_ installer.
+2. After downloading the .exe file, double click it. A window will appear and guide you through the process of installing KA Lite on the server.
+
+Mac Installation
+================
+
+1. Download the KA Lite `OSX installer <https://disney.com>`_.
+
+.. warning:: Dear maintainers, please update this link.
+
+Linux: Debian/Ubuntu Installation
+=================================
+
+1. Download `the deb file <http://overtag.dk/upload/ka-lite_0.14~dev9-1_all.deb>`_ and enjoy!
+
+.. _linux-pypi-install:
+
+Linux: Installing as a PyPI package
+===================================
+
+.. note:: Type the following commands in a terminal.
+
+#. Install pip (*sudo apt-get install python-pip* or use your distro's package manager).
+#. (Recommended; essential on slower platforms like Raspberry Pi) Install M2Crypto (*sudo apt-get install python-m2crypto*).
+#. Run *sudo pip install ka-lite-static*.
+#. Run *kalite manage setup*. Follow the on-screen prompts to complete the setup.
+#. After completing the setup, follow the on-screen instructions to start KA Lite using the *kalite* command!
+

```

Comment:
I promise both of these comments are addressed. I don't know why this is still showing up.